core/src/main/java/org/geysermc/geyser/skin/SkinProvider.java

```diff
@@ -260,42 +259,27 @@ static CompletableFuture<SkinProvider.SkinData> requestSkinData(PlayerEntity ent
                             cape = getCachedBedrockCape(entity.getUuid());
                         }
 
-                        if (cape.failed() && ALLOW_THIRD_PARTY_CAPES) {
-                            cape = getOrDefault(requestUnofficialCape(
-                                    cape, entity.getUuid(),
-                                    entity.getUsername(), false
-                            ), EMPTY_CAPE, CapeProvider.VALUES.length * 3);
-                        }
-
-                        boolean isDeadmau5 = "deadmau5".equals(entity.getUsername());
-                        // Not a bedrock player check for ears
-                        if (geometry.failed() && (ALLOW_THIRD_PARTY_EARS || isDeadmau5)) {
-                            boolean isEars;
-
-                            // Its deadmau5, gotta support his skin :)
-                            if (isDeadmau5) {
-                                isEars = true;
-                            } else {
-                                // Get the ears texture for the player
-                                skin = getOrDefault(requestUnofficialEars(
-                                        skin, entity.getUuid(), entity.getUsername(), false
-                                ), skin, 3);
-
-                                isEars = skin.isEars();
+                        // Call event to allow extensions to modify the skin, cape and geo
+                        boolean isBedrock = GeyserImpl.getInstance().connectionByUuid(entity.getUuid()) != null;
+                        final SkinData[] skinData = {new SkinData(skin, cape, geometry)};
```

Comment:
...Why is this an array???

Reply:
![image](https://github.com/GeyserMC/Geyser/assets/5401186/beee5421-4115-43fc-8817-1d38b5be7ec2)